src/index.js

```diff
@@ -0,0 +1,54 @@
+import './scss/style.scss';
+import searchIcon from './images/search-icon.png';
+
+let weather = {
+  apiKey: 'c20d664fe15511ff6b176f9e20df512f',
+  fetchweather: function (city) {
```

Comment:
Let us use the convention for defining anonymous function with es6, with the use of arrow function this should look like this.
`fetchweather: (city)=> {`